### ISSUE TITLE
Commit hash always 'unknown' 

### ISSUE THREAD

**Issue by CodeFinder2** [danielwelch/hassio-zigbee2mqtt#146]
I am using the latest zigbee2mqtt with this HassIO addon and observed that it always publishes something like
```
AM MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"version":"1.3.0","commit":"unknown","coordinator_firmware":20190109,"log_level":"debug","permit_join":false}'
```

I am referring to the `"commit":"unknown"` here. Should be just a minor issue. Or do I miss something?

**Comment by paskyecijano** [comment]
I have the same error

![image](https://user-images.githubusercontent.com/3083124/56029624-189db680-5d1b-11e9-8fcf-7185bf6be03f.png)

**Comment by hawaltie** [comment]
I also have the same 'error'

**Comment by danielwelch** [comment]
Looking at merging this in for version 1.5.0

**Comment by ciotlosm** [comment]
It still shows unknown in 1.6.0

```
  zigbee2mqtt:info 10/20/2019, 9:44:05 AM Logging to directory: '/share/zigbee2mqtt/log/2019-10-20.09-44-02'
  zigbee2mqtt:info 10/20/2019, 9:44:05 AM Starting zigbee2mqtt version 1.6.0 (commit #unknown)
```

```
  zigbee2mqtt:info 10/20/2019, 9:44:12 AM MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
  zigbee2mqtt:info 10/20/2019, 9:44:12 AM MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"version":"1.6.0","commit":"unknown","coordinator":20190608,"log_level":"info","permit_join":false}'
```

**Comment by danielwelch** [comment]
@ciotlosm yeah, that's expected. Here's the relevant bit in the edge version Dockerfile https://github.com/danielwelch/hassio-zigbee2mqtt/blob/0f9046134f98a10c50bf404e9ca0664747bb97d7/zigbee2mqtt-edge/Dockerfile#L25
I didn't include this in stable version because it's using a known release of zigbee2mqtt by definition, which I figured would be the most relevant info. With that said, that release obviously is tied to a commit, and I guess it couldn't hurt to provide the hash as well, even if it is kind of redundant.

**Comment by CodeFinder2** [comment]
> @ciotlosm yeah, that's expected. Here's the relevant bit in the edge version Dockerfile
> 
> https://github.com/danielwelch/hassio-zigbee2mqtt/blob/0f9046134f98a10c50bf404e9ca0664747bb97d7/zigbee2mqtt-edge/Dockerfile#L25
> 
> 
> I didn't include this in stable version because it's using a known release of zigbee2mqtt by definition, which I figured would be the most relevant info. With that said, that release obviously is tied to a commit, and I guess it couldn't hurt to provide the hash as well, even if it is kind of redundant.

So, will it finally be included? (I think having it printed is useful, although redundant. Alternatively, you should remove the commit hash completely for releases because this now looks a bit 'incomplete'. 🙈 Nonetheless, I would appreciate seeing the version number and commit hash.) 

It's still not in v1.7.1:

> zigbee2mqtt:info  2019-12-11T13:23:46: Starting zigbee2mqtt version 1.7.1 (commit #unknown)

>

**Comment by ciotlosm** [comment]
@danielwelch should we close this based on https://github.com/danielwelch/hassio-zigbee2mqtt/pull/388 ?

**Comment by danielwelch** [comment]
If someone can confirm it works :)

**Comment by danielwelch** [comment]
So you're not on edge? This is only supposed to work on edge as of now.

**Comment by CodeFinder2** [comment]
Ah sry, yes didn't use edge 🙈

**Comment by ciotlosm** [comment]
Finally got to install a dev instance wtih a separate stick. Seems the commit hash is still not working for edge. 
```
Zigbee2MQTT:info  2020-09-19 07:41:47: Starting Zigbee2MQTT version 1.14.4-dev (commit #unknown)
```

**Comment by ciotlosm** [comment]
@danielwelch apparently there is an [error](https://dev.azure.com/danielwelch2101/hassio-zigbee2mqtt/_build/results?buildId=424&view=logs&j=d822dde3-66a5-599f-9c9e-ed09ffe6609f&t=22f3bdd9-8b84-57c5-833f-5accb3b8a54d&l=1531) when running the pipeline.

**Comment by ciotlosm** [comment]
Fixed in 1.14.4.2
